src/libraries/System.Formats.Tar/tests/TarFile/TarFile.CreateFromDirectory.File.Tests.cs

```diff
@@ -253,5 +255,57 @@ public void SkipRecursionIntoBaseDirectorySymlink()
 
             Assert.Null(reader.GetNextEntry());
         }
+
+        [ConditionalTheory]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        [MemberData(nameof(GetPaxAndGnuTestCaseNames))]
+        public void ResultingArchive_CanBeExtractedByOtherTools(string testCaseName)
+        {
+            if (testCaseName == "specialfiles" && !PlatformDetection.IsSuperUser)
+            {
+                throw new SkipTestException("specialfiles needs sudo permissions for extraction");
+            }
+
+            using TempDirectory root = new TempDirectory();
+
+            string archivePath = GetTarFilePath(CompressionMethod.Uncompressed, TestTarFormat.gnu.ToString(), testCaseName);
+            string tarExtractedPath = Path.Join(root.Path, "tarExtracted");
+            Directory.CreateDirectory(tarExtractedPath); // rename to gnuTarExtractedPath et al
```

Comment:
```cs
// rename to gnuTarExtractedPath et al
```
Do you mind explaining this comment? The name used in the previous line is `tarExtracted`.

Reply:
It was there to remind me to rename the folder/variable to something that alludes to "GNU Tar" but I then discovered that macOS' tar utility is named BSD Tar so I then decided to just name it tar*, and I forgot to remove the comment.